modules/components/Navbar/NavigationBar.tsx

```diff
@@ -8,15 +8,16 @@ import { authenticatedUserSelector } from "@/shared/redux/reducers/user.reducer"
 import { ERole } from "@/shared/typedefs";
 
 import classes from "./NavigationBar.module.css";
+import { INavigationBarProps } from "./NavigationBar.types";
 
-const NavigationBar = () => {
+const NavigationBar = ({ open }: INavigationBarProps) => {
   const user = useAppSelector(authenticatedUserSelector);
   return (
     <Box h={"9vh"} bg={"#011733"} className={classes["navlinks"]}>
       <Text>Superteacher</Text>
       <Flex justify={"center"} align={"center"} gap={15}>
         <Text className={classes["dashboard"]}>Dashboard</Text>
-        {user.claim === ERole.TEACHER && <FaPlus />}
+        {user.claim === ERole.TEACHER && <FaPlus onClick={open} />}
```

Comment:
```suggestion
        {user.claim === ERole.TEACHER ? <FaPlus onClick={open} />: null}
```

[When conditionally rendering React components, let's use ternary and avoid the && operator](https://www.crocoder.dev/blog/react-conditional-rendering/#:~:text=However%2C%20there%20is%20a%20catch,render%20exactly%20what%20you%20expect.)